applications/openshift/kubelet/kubelet_anonymous_auth/rule.yml

```diff
@@ -6,8 +6,10 @@ platform: {{{ product }}}-node
 
 {{%- if product == "eks" %}}
 {{%- set kubeletconf_path = "/etc/kubernetes/kubelet/kubelet-config.json" %}}
+{{%- set platform_noun = "EKS" %}}
```

Comment:
Instead of defining the `platform_noun` in every rule, it should be defined in the products's `product.yml` file.
There is already `full_name` defined  that could be leveraged, if that is too long maybe a new `short_name` variable would make sense?
https://github.com/ComplianceAsCode/content/blob/master/products/eks/product.yml#L2

Reply:
Hi @rhmdnd , do you plan to continue working on this sooner or later?

By the way, `short_name` would make sense, IMO.

Reply:
@rhmdnd any updates?